.gitignore

```diff
@@ -1,4 +1,7 @@
-wordpress
-data
+app
```

Comment:
Most frameworks already uses `app` dir for specific codes. e.g. https://laravel.com/docs/5.5/structure

If we use this then users will have to `cd ..../app/app/....`.

So can we think of an alternative? e.g. `web`, `source`

Reply:
or maybe keep `wordpress` for now 😃